src/native/libs/System.Security.Cryptography.Native/entrypoints.c

```diff
@@ -145,11 +145,12 @@ static const Entry s_cryptoNative[] =
     DllImportEntry(CryptoNative_EvpDigestSqueeze)
     DllImportEntry(CryptoNative_EvpDigestUpdate)
     DllImportEntry(CryptoNative_EvpDigestXOFOneShot)
-    DllImportEntry(CryptoNative_EvpMacFetch)
-    DllImportEntry(CryptoNative_EvpMacFree)
+    DllImportEntry(CryptoNative_EvpMacCtxDup)
```

Comment:
Some of this was just sorting. The only meaningful change here was adding MacCtxDup and HmacCopy.